pkg/proto/configuration/global/global.proto

```diff
@@ -294,6 +295,16 @@ message Configuration {
   // Keys of this map correspond to the suffixes of the RLIMIT_*
   // constants (e.g., "NOFILE" to limit the number of file descriptors).
   map<string, SetResourceLimitConfiguration> set_resource_limits = 15;
+
+  // List of compression algorithms supported by the Content Addressable
```

Comment:
This has no business being part of pkg/global. That package is used by all Buildbarn binaries. Announcing compressors is specific to bb_storage.